src/noggit/scripting/script_context.cpp

```diff
@@ -0,0 +1,38 @@
+// This file is part of Noggit3, licensed under GNU General Public License (version 3).
+#include <noggit/scripting/script_context.hpp>
+#include <noggit/scripting/scripting_tool.hpp>
+#include <noggit/World.h>
+#include <noggit/camera.hpp>
+
+namespace noggit
+{
+  namespace scripting
+  {
+    // makes it easier for scripters to not pass this around
+    static script_context* ctx;
+
+    void set_ctx(script_context* nctx)
+    {
+      ctx = nctx;
```

Comment:
ctx never gets free'd. current context only gets released because it runs out of scope in scripting_tool::sendUpdate